src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -157,7 +157,7 @@ def _add_model_args(parser):
 
 def _add_profile_args(parser):
     profile_group = parser.add_argument_group("Profiling")
-    load_management_group = profile_group.add_mutually_exclusive_group(required=True)
+    load_management_group = profile_group.add_mutually_exclusive_group(required=False)
```

Comment:
The idea being that we let PA fall back to its default behavior here?

Reply:
Yep, exactly.

Reply:
yea this is better than i was thinking of doing it.
nice job!